frontend/packages/client/package.json

```diff
@@ -27,18 +27,10 @@
     "react-error-boundary": "^3.1.4",
     "react-hook-form": "^7.11.0",
     "react-images-upload": "^1.2.8",
-    "react-redux": "^7.2.4",
+    "react-query": "^3.39.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "redux": "^4.1.0",
-    "redux-persist": "^6.0.0",
-    "redux-persist-cookie-storage": "^1.0.0",
-    "redux-thunk": "^2.3.0",
-    "sass": "^1.49.8",
-    "string-strip-html": "8.3.0",
-    "prettier": "^2.7.1",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0"
```

Comment:
"react-redux": "^7.2.4", => removed 
    "redux": "^4.1.0", => removed 
    "redux-persist": "^6.0.0",  => removed 
    "redux-persist-cookie-storage": "^1.0.0",  => removed 
    "redux-thunk": "^2.3.0",  => removed 
    "sass": "^1.49.8",  => moved to dev dependencies, only used at build time 
    "prettier": "^2.7.1",  => moved to dev dependencies
    "eslint-config-prettier": "^8.5.0"  => moved to dev dependencies https://github.com/prettier/eslint-config-prettier#installation
    "eslint-plugin-prettier": "^4.0.0"  => moved to dev dependencies https://github.com/prettier/eslint-plugin-prettier#installation

Reply:
@germanurrus prettier and the eslint config have to be here otherwise the staging build breaks because of how monorepos work

Reply:
added deps back and left out redux